code/__defines/configuration.dm

```diff
@@ -0,0 +1,7 @@
+//config files
+#define CONFIG_GET(X) global.config.Get(/datum/config_entry/##X)
```

Comment:
> make these GET_CONFIG and SET_CONFIG, mass replace everywhere

I think that in syntaxis better CPNFIG_GET, because CONFIG is system GET is proc. You don't write destroy.mob in the code.